What's new in: "test/MsSQLTest.hs"
hunk ./test/MsSQLTest.hs 32
-  , dbl :: Maybe ByteString -- TODO: FIXME
-  , datec :: Maybe Day
+  , dbl :: Maybe ByteString -- TODO: FIXME [_$_]
+  , datec :: Maybe Day {-
hunk ./test/MsSQLTest.hs 37
-  , uuid :: Maybe UUID
+  , uuid :: Maybe UUID -}
hunk ./test/MsSQLTest.hs 54
-
-unit_listCompare :: IO ()
-unit_listCompare = pure ()
-
-testConnectInfo = connectInfo "Driver={ODBC Driver 17 for SQL Server};Server=localhost;Database=hask;UID=sa;PWD=P@ssw0rd;"
+testConnectInfo = connectInfo "Driver={ODBC Driver 17 for SQL Server};Server=localhost;Database=hask;UID=sa;PWD=P@ssw0rd;ApplicationIntent=ReadOnly"
+
hunk ./test/MsSQLTest.hs 59
+  print con
hunk ./test/MsSQLTest.hs 61
-
-
-unit_sqldirect :: IO ()
-unit_sqldirect = do
-  Right con <- connect $ testConnectInfo
-  sqldirect1 con "select * from test1"
-  disconnect con
-
-
-unit_sqlcolbind :: IO ()
-unit_sqlcolbind = do
-  Right con <- connect $ testConnectInfo
-  Right res <- query con "select * from test1" :: IO (Either SQLErrors (Vector TestT))
-  print (res)
---  mapM ((maybe (pure ()) T.putStrLn) . col4) res
-  [_$_]
-  putStrLn $ ordT $ Foo 1 2 4 '3' True
-  --disconnect con
-
-unit_sqlgetinfo :: IO ()
-unit_sqlgetinfo = do
-  Right con <- connect $ testConnectInfo
-  res <- sqlGetInfo con 1
-  res @?= "1"
-  print ("INTSZZZZZZZZZZZZZ" :: String, intsz)
-  disconnect con
-  [_$_]
-
-unit_sqlupdate :: IO ()
-unit_sqlupdate = do
-  Right con <- connect $ testConnectInfo
-  Right res <- execute con "update test1 set dtz = SYSUTCDATETIME() where test_id<>0"
-  print res
---  disconnect con
-
-
-unit_sqlinsert :: IO ()
-unit_sqlinsert = pure ()
-{-unit_sqlinsert = do
-  Right con <- connect $ testConnectInfo
-  Right res <- execute con "insert test1 (test_id, col1) values (4,5)"
-  print res
-  disconnect con  [_$_]
--}
+  print "Disconnected"
